README.md

```diff
@@ -1,41 +1,112 @@
+# Migrating receipts to RevenueCat via bulk import
```

Comment:
This guide is turning into documentation so a title might help discoverability and scannability